package.json

```diff
@@ -19,7 +19,8 @@
     "Jack Domleo <jackdomleo7@gmail.com> (https://github.com/jackdomleo7)",
     "Dan Spratling <dan@danspratling.dev> (https://github.com/danspratling)",
     "Savvas Stephanides (https://github.com/SavvasStephanides)",
-    "Veeresh (https://github.com/veerreshr)"
+    "Veeresh (https://github.com/veerreshr)",
+    "Devin Ford (https://github.com/devindford"
```

Comment:
Ah Devin, sorry I missed this earlier! You're missing a closing bracket/brace `)` here. 😊